src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LspLogger.cs

```diff
@@ -58,6 +58,10 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             _ => throw new NotImplementedException(),
         };
         var message = formatter(state, exception);
+        if (message == "[null]" && exception is not null)
```

Comment:
Snuck this one in, just because I saw it come up in the logs, and it wasn't exactly helpful. Not sure if its something on my machine, but whatever message formatter we're being passed here doesn't seem to use the `exception` at all.